package.json

```diff
@@ -35,5 +37,8 @@
   "lint-staged": {
     "src/**/*.js": "eslint --fix",
     "src/**/*": "prettier --write"
+  },
+  "dependencies": {
+    "netlify-lambda": "2.0.1"
```

Comment:
I think this should go in `devDependencies` since it's only used during build, not during site runtime

Reply:
Fixed!